cluster/cluster.py

```diff
@@ -217,7 +217,9 @@ def deploy_etcd_cluster(hosted_zone, etcd_bucket, region):
             bucket.create(CreateBucketConfiguration={'LocationConstraint': region})
             bucket.wait_until_exists()
 
-    subprocess.check_call(['senza', 'create', 'etcd-cluster.yaml', 'etcd', 'HostedZone={}'.format(hosted_zone), 'EtcdS3Backup={}'.format(etcd_bucket)])
+    subprocess.check_call(['senza', 'create', '--update-if-exists', 'etcd-cluster.yaml', 'etcd',
```

Comment:
I think it doesn't matter anyway as `has_etcd_cluster()` has to be `false` to trigger it.